From kalle.valo@iki.fi  Sun Jan 25 14:26:05 2009
From: Kalle Valo <kalle.valo@iki.fi>
Date: Sun, 28 Dec 2008 20:32:33 +0200
Subject: Staging: at76_usb: mention mac80211 port in TODO file
To: gregkh@suse.de
Cc: linux-wireless@vger.kernel.org
Message-ID: <20081228183233.15512.10965.stgit@tikku>


at76_usb has actually been already ported to use mac80211. Update the TODO
file to reflect this.

Signed-off-by: Kalle Valo <kalle.valo@iki.fi>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/at76_usb/TODO |    9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

--- a/drivers/staging/at76_usb/TODO
+++ b/drivers/staging/at76_usb/TODO
@@ -1,2 +1,7 @@
-rewrite the driver to use the proper in-kernel wireless stack
-instead of using its own.
+Fix the mac80211 port of at76_usb (the proper in-kernel wireless
+stack) and get it included to the mainline. Patches available here:
+
+http://git.kernel.org/?p=linux/kernel/git/linville/wireless-legacy.git;a=shortlog;h=at76
+
+Contact Kalle Valo <kalle.valo@iki.fi> and linux-wireless list
+<linux-wireless@vger.kernel.org> for more information.
